internal/config/config.go

```diff
@@ -49,6 +50,11 @@ type DebugConfig struct {
 	ListenPort    uint   `yaml:"port" envconfig:"DEBUG_PORT"`
 }
 
+type BasicAuth struct {
+	Username string `yaml:"username" envconfig:"BASIC_AUTH_USERNAME"`
+	Password string `yaml:"password" envconfig:"BASIC_AUTH_PASSWORD"`
+}
```

Comment:
Remove this. Any configuration should be reserved for the correct plugin. This should be configured in the plugin's configuration.